src/orders/components/OrderFulfillStockExceededDialogLine/OrderFulfillStockExceededDialogLine.tsx

```diff
@@ -14,13 +14,17 @@ import { useStyles } from "../OrderFulfillStockExceededDialog/styles";
 export interface OrderFulfillStockExceededDialogLineProps {
   line: OrderFulfillLineFragment | FulfillmentFragment["lines"][0];
   warehouseId: string;
-  formsetData: OrderFulfillStockInputFormsetData;
+  formsetData: OrderFulfillStockFormsetData;
 }
 
 const OrderFulfillStockExceededDialogLine: React.FC<OrderFulfillStockExceededDialogLineProps> = props => {
   const { line: genericLine, warehouseId, formsetData } = props;
 
-  const line = "orderLine" in genericLine ? genericLine.orderLine : genericLine;
+  const line = genericLine
+    ? "orderLine" in genericLine
+      ? genericLine.orderLine
+      : genericLine
+    : null;
```

Comment:
I think you can just move 

```
  if (!genericLine) {
    return null;
  }
```
from lines 34-36 up here and then we can avoid nested ternary